src/main/java/gvs/business/logic/LayoutController.java

```diff
@@ -2,173 +2,198 @@
 import gvs.interfaces.IEdge;
 import gvs.interfaces.IVertex;
 
 /**
- * Creates and prepares the elements which have to be layouted.
+ * Creates and prepares the elements which need to be layouted.
  * 
- * @author aegli
+ * Executes a separate ticker thread which calls the LayoutController in a
+ * certain interval.
+ * 
+ * @author mwieland
  *
  */
 @Singleton
-public class LayoutController extends Observable implements Tickable {
-
-  private boolean doSoftLayout = false;
-  private Area area = null;
-  private AreaTicker ticker = null;
-  private Particle particle = null;
-  private Random random = null;
+public class LayoutController implements Tickable {
 
-  private Collection<IVertex> vertizes = null;
-  private Collection<IEdge> edges = null;
+  private Area area;
+  private AreaTicker ticker;
+  private Random random;
 
-  private final int setLayoutStableAfterTime = 10000;
-
-  private static final int DEFAULT_RADIUS = 40/* radius */;
-  private static final int DEFAULT_MASS = 50/* masse */;
-  private static final int SOFT_MULTIPLIER = 100;
+  private static final int DEFAULT_MASS = 40;
+  private static final int SOFT_MULTIPLIER = 10;
   private static final int FIXED_MULTIPLIER = 10;
-  private static final int DEFAULT_DISTANCE = 70;
+  private static final int DEFAULT_DISTANCE = 300;
   private static final int DEFAULT_IMPACT = 10;
-  private static final int DEFAULT_AREA_Y = 900;
-  private static final int DEFAULT_AREA_X = 950;
+
+  private static final int MAX_LAYOUT_DURATION_MS = 10000;
+  private static final int DEFAULT_TICK_RATE = 50;
+
+  private static final int DEFAULT_AREA_HEIGHT = 1300;
+  private static final int DEFAULT_AREA_WIDTH = 1600;
+
   private static final int DEFAULT_SEED = 4000;
-  private static final int DEFAULT_RATE = 40;
 
   private static final Logger logger = LoggerFactory
       .getLogger(LayoutController.class);
 
+  public LayoutController() {
+    this.area = new Area(
+        new AreaDimension(DEFAULT_AREA_WIDTH, DEFAULT_AREA_HEIGHT));
+    this.random = new Random(DEFAULT_SEED);
+
+    initializeLayoutGuard();
+  }
+
   /**
-   * Starts layout engine.
-   *
+   * Initializes the guard, which protects the layouter from running Infinitely.
    */
-  public LayoutController() {
-    vertizes = new Vector<>();
-    edges = new Vector<>();
-    area = new Area(new AreaDimension(DEFAULT_AREA_X, DEFAULT_AREA_Y));
-    ticker = new AreaTicker(this, DEFAULT_RATE);
-    ticker.start();
-    logger.info("Starting graph layout controller");
-    logger.debug("Starting layout guard");
-    Timer guard = new Timer();
+  private void initializeLayoutGuard() {
+    boolean isDaemon = false;
+    Timer guard = new Timer(isDaemon);
     LayoutGuard layoutGuard = new LayoutGuard(area);
-    guard.schedule(layoutGuard, setLayoutStableAfterTime);
-    random = new Random();
-    random.setSeed(DEFAULT_SEED);
+    guard.schedule(layoutGuard, MAX_LAYOUT_DURATION_MS);
   }
 
   /**
```

Comment:
JavaDoc entsprcht nicht den gesetzten Regeln:
Sagt was die Funktion tun sollte
-> "layout the received vertivces" o.ä.

Reply:
vllt könnte man noch erklären, was soft Points / soft layout genau bedeutet